src/base/Raw.h

```diff
@@ -42,16 +49,17 @@ class Raw
     /// Minimum section debugging level necessary for printing. By default,
     /// small strings are always printed while large strings are only printed
     /// if DBG_DATA debugging level is enabled.
-    int level;
+    int level = -1;
 
 private:
     void printHex(std::ostream &os) const;
 
-    const char *label_; ///< optional data name or ID; triggers size printing
-    const char *data_; ///< raw data to be printed
-    size_t size_; ///< data length
-    bool useHex_; ///< whether hex() has been called
-    bool useGap_; ///< whether to print leading space if label is missing
+    const char *label_ = nullptr; ///< optional data name or ID; triggers size printing
+    const void *data_ = nullptr; ///< raw data to be printed
+    size_t size_ = 0; ///< data length
+    size_t printableSize_ = 256; ///< do not print more data
+    bool useHex_ = false; ///< whether hex() has been called
```

Comment:
Please address `ErrorPage::BuildErrorPrinter::print()` TODO that was waiting for this "Add support for prefix printing to Raw" PR.